game-app/game-core/src/main/java/games/strategy/triplea/odds/calculator/AggregateResults.java

```diff
@@ -13,30 +13,79 @@
 import java.util.Optional;
 import lombok.Getter;
 import lombok.Setter;
+import org.apache.commons.math3.stat.descriptive.moment.Mean;
+import org.apache.commons.math3.stat.descriptive.moment.StandardDeviation;
 import org.triplea.java.collections.IntegerMap;
 import org.triplea.util.Tuple;
 
-/** A container for the results of multiple battle simulation runs. */
+/**
+ * A container for the results of multiple battle simulation runs.
+ *
+ * <p>This class wraps a collection of {@code BattleResult}s and provides methods to query certain
+ * statistical properties, e.g. the win probability, or the standard deviation for the number of
+ * unit left.
+ *
+ * <p>This class does not restrict the battle result to come from the same battle setup. If this is
+ * desired, the user must ensure that this property hold.
+ *
+ * <p>Note on bias correction: Given a set of data points there are different variances (or standard
```

Comment:
Nit: Please check if you can shorten up this javadoc with a second edit.
EG: "For more information see any book about statistics or, can just be shorted to", "see: <url>"

Another point of view, see how many words you can take out while still retaining the original meaning. Ditto for any commentary and blocks of text added.

Site-note: great to see the documentation added :+1: